### PR TITLE
Layout implement docs template

### PR DESCRIPTION
**Pull request by Fercas123**

### CODE REVIEW

File: packages/core/src/layout/GridLayout/GridLayout.css [hunk 1]
```diff
@@ -2,12 +2,14 @@
   --grid-layout-gap-density-multiplier: 2;
 }
 
+/* Default variables applied to the root element */
 .uitkGridLayout {
   --grid-layout-space: 1fr;
   --grid-layout-row-gap: var(--grid-layout-gap-density-multiplier, 3);
```
Comment (libertymayc) [inline packages/core/src/layout/GridLayout/GridLayout.css:8]: when it's a css attribute like row-gap, make it camel case -rowGap

File: .changeset/beige-islands-kick.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@jpmorganchase/uitk-core": minor
```
Comment (origami-z) [inline .changeset/beige-islands-kick.md:2]: If only internal var and doc being changed, then only patch is needed?

**Review by JoanaMMoreira — APPROVED** [review]
Looks really good :)

File: packages/core/stories/layout/flex-layout.doc.stories.mdx [hunk 3]
```diff
@@ -38,29 +46,30 @@ import { FlexLayout, FlexItem } from "@jpmorganchase/uitk-lab";
...
 
+### Flex Layout with separators
+
+<Canvas>
+  <Story id="core-layout-flexlayout--flex-layout-with-separators" />
+</Canvas>
+
 ### Responsive
 
 The flex layout supports the use of responsive props, so you can defined different behaviours based on viewport size.
```
[the remark this answers is not in the document]
Comment (Fercas123) [reply packages/core/stories/layout/flex-layout.doc.stories.mdx:75]: I'll make it 6

**Pull request merged**